app/src/common/urls.js

```diff
@@ -109,6 +109,10 @@ export const URLS = {
       'page.sort': 'time,ASC',
     })}`,
   logItemActivity: (activeProject, itemId) => `${urlBase}${activeProject}/activity/item/${itemId}`,
+  logItemStackTrace: (activeProject, itemId) =>
+    `${urlBase}${activeProject}/log?filter.eq.item=${itemId}&page.page=1&page.size=1&filter.in.level=ERROR&page.sort=time%2CDESC`,
+  logItemStackTraceMessageLocation: (activeProject, itemId, stackTraceItemId, pageSize, level) =>
+    `${urlBase}${activeProject}/log/${stackTraceItemId}/page?filter.eq.item=${itemId}&page.page=1&page.size=${pageSize}&filter.gte.level=${level}&page.sort=time%2CASC`,
```

Comment:
Please use getQueryParams  func to make params more readable.

Reply:
Done